sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_28_days_later_v1/query.sql

```diff
@@ -0,0 +1,75 @@
+WITH clients_first_seen_28_days_ago AS (
+  SELECT
+    client_id,
+    sample_id,
+    country AS country_code,
+    normalized_channel AS channel,
+    app_build_id AS build_id,
+    normalized_os AS os,
+    mozfun.norm.truncate_version(normalized_os_version, "minor") AS os_version,
+    attribution_source,
+    distribution_id,
+    attribution_ua,
+    first_seen_date AS date,  -- TODO: Why rename this?
```

Comment:
question for @irrationalagent

Reply:
we probably don't need to rename it for this table. I think its renamed in the NtM query because date ends up being a join key for when we assemble to the funnel for looker. probably more clear to leave it named as `first_seen_date` here.